src/ansible/roles/ipasmartcard/tasks/main.yml

```diff
@@ -0,0 +1,76 @@
+- name: Checking DNS resolution first
+  shell: |
+    dig ipa-ca.{{ ipa_domain }} | grep NOERROR
+  register: result
+  until: result.rc == 0
+  retries: 10
+  delay: 10
```

Comment:
I wonder if the delay can be shorter (1 ~ 2s). How much time does this step usually take to settle down?

Reply:
I'd have to dig back through some old failures but, I thought it was almost 30 seconds but, considering that there are other steps between when the IPA server is setup and this, that probably pads a little extra time.   So, I'll switch this to 2 seconds and we can adjust to higher only if needed?